pandas/tests/frame/test_alter_axes.py

```diff
@@ -871,6 +871,29 @@ def test_rename_bug2(self):
                              columns=["a"])
         tm.assert_frame_equal(df, expected)
 
+    def test_rename_errors(self):
+        # GH 13473
+        # rename now works with errors parameter
+
+        # Error has to be thrown and is thrown
+        df = DataFrame(columns=['A', 'B', 'C', 'D'])
+        with pytest.raises(KeyError):
```

Comment:
Can you also add expected message here?

Reply:
@TomAugspurger My bad, I normally do so when going through the feedback so that I know which ones I have fixed, after which I push my changes. Ofcourse, because I still had these questions, I now did not yet push the changes and hence they are marked resolved but not resolved in the repo.

Could you give me an example of the standard used syntax for test functions, as described by William? I couldn't find any tests which are appended with `_tests`

Reply:
I'm not sure what @WillAyd was referring to in 
https://github.com/pandas-dev/pandas/pull/25535/files#r262267286. I'm guessing he'll clarify.